packages/svelte/src/compiler/utils/fuzzymatch.js

```diff
@@ -19,18 +19,20 @@ const GRAM_SIZE_UPPER = 3;
  * @param {string} str2
  */
 function _distance(str1, str2) {
-	if (str1 === null && str2 === null) {
+	if (!str1 && !str2) {
 		throw 'Trying to compare two null values';
 	}
-	if (str1 === null || str2 === null) return 0;
+
+	if (!str1 || !str2) {
+		return 0
+	}
+
 	str1 = String(str1);
 	str2 = String(str2);
+
 	const distance = levenshtein(str1, str2);
-	if (str1.length > str2.length) {
-		return 1 - distance / str1.length;
-	} else {
-		return 1 - distance / str2.length;
-	}
+
+	return 1 - distance / (str1.length > str2.length ? str1.length : str2.length);
```

Comment:
I’d favor if-else over ternary. If you must change this, `Math.max(str1.length, str2.length)` is better.